src/components/WebexParticipantRoster/WebexParticipantRoster.jsx

```diff
@@ -5,15 +5,23 @@ import {List} from '@momentum-ui/react';
 import WebexParticipant from '../WebexParticipant/WebexParticipant';
 import useMemberships from '../hooks/useMemberships';
 
+/**
+ * Enum for types of destinations.
+ *
+ * @external DestinationType
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MembershipsAdapter.js#L21}
+ */
+
 /**
  * Displays the roster of Webex meeting or room participants.
  *
- * @param {string} props.location  the roster of Webex meeting or room participants.
+ * @param {string} props.destinationID  the roster of Webex meeting or room participants.
```

Comment:
From the interface
```suggestion
 * @param {string} props.destinationID  ID of the destination for which to get members
```

I know you just moved it but since we're improving things...